### PR TITLE
AEA-1574 add log info message if loaded component is abstract

### LINKED ISSUE #2474: Log if abstract skills is used in agent
**Is your feature request related to a problem? Please describe.**
Added abstract skill and it does not work (that obvious), but hard to find out why.

**Describe the solution you'd like**
Just add info log that abstract skill loaded and will be not used by the agent

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Add log info message if loaded component is abstract

## Fixes

Fixes #2474

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: aea/aea_builder.py [hunk 1]
```diff
@@ -1918,6 +1918,10 @@ def _load_and_add_components(
                 new_configuration = self._overwrite_custom_configuration(configuration)
                 if new_configuration.is_abstract_component:
                     load_aea_package(configuration)
+                    self.logger.info(
```
Comment (marcofavorito) [inline aea/aea_builder.py:1921]: do we agree on log level? Maybe debug?

File: tests/test_aea_builder.py [hunk 2]
```diff
@@ -596,16 +596,36 @@ def test_set_from_config_custom():
...
     builder = AEABuilder()
     builder.set_name("aea_1")
     builder.add_private_key("fetchai")
 
-    skill = Skill.from_dir(dummy_skill_path, Mock(agent_name="name"))
-    skill.configuration.is_abstract = True
-    builder.add_component_instance(skill)
-    builder._load_and_add_components(
-        ComponentType.SKILL, Resources(), "aea_1", agent_context=Mock(agent_name="name")
-    )
```
Comment (marcofavorito) [inline tests/test_aea_builder.py:608]: I think this test wasn't testing properly (no assert)

File: aea/aea_builder.py (hunk 1 above)
Comment (DavidMinarsch) [reply aea/aea_builder.py:1921]: Debug please!

**Review by solarw — APPROVED** [review]
LGTM

**Pull request merged**